pandas/tests/test_generic.py

```diff
@@ -1853,6 +1853,81 @@ def test_to_xarray(self):
                            expected,
                            check_index_type=False)
 
+    def test_getitem_index(self):
+        # GH8162
```

Comment:
you need to see what this does on Series and what the failure cases are (e.g. duplicated names), does it raise errors anywhere, what?